assets/css/3-sections/_header.sass

```diff
@@ -36,7 +36,7 @@ header
     color: white
     text-transform: uppercase
     letter-spacing: 0.05em
-    margin-bottom: 20px
+    margin-bottom: 20px !important
```

Comment:
Because it is difficult to overwrite `!important`s in modular CSS. You end up overwriting with another `!important`. So the general rule is to use `!important`only for helper classes and not use it to overwrite stuff with higher specificity.